src/types/api/schema/replication.ts

```diff
@@ -0,0 +1,94 @@
+import type {IssueMessage} from '../query';
+
+import type {TPathID} from './shared';
+
+/**
+ * source: https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/replication.proto
+ */
+interface TStaticCredentials {
+    User?: string;
+    Password?: string;
+    PasswordSecretName?: string;
+}
+
+/**
+ * source: https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/replication.proto
+ */
+interface TOAuthToken {
+    Token?: string;
+    TokenSecretName?: string;
+}
+
+/**
+ * source: https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/replication.proto
+ */
+export interface TConnectionParams {
+    Endpoint?: string;
+    Database?: string;
+    StaticCredentials?: TStaticCredentials;
+    OAuthToken?: TOAuthToken;
+}
+
+interface TReplicationConfigTTargetEverything {
+    DstPrefix?: string;
+}
+
+export interface TReplicationConfigTTargetSpecificTTarget {
+    SrcPath?: string;
+    DstPath?: string;
+}
+
+interface TReplicationConfigTTargetSpecific {
```

Comment:
I wanted to use naming like `TReplicationConfig_TTargetSpecific_TTarget` here, as all the proto generators do, but come across ESLint. Dunno how to do it better, the current name is difficult to read.

Reply:
It does not ensures the uniqueness of names.

Reply:
Simplified.